ug/instructors.md

```diff
@@ -56,3 +77,12 @@ The repositories used for initial bug reporting will be created automatically by
   This must be a public repo with write access given to students.
 * A repo to hold bugs for tutors to moderate e.g., `module-org/pe-moderation`<br>
   This can be a private repo with write access given to tutors.
+  
+
+## Creating a session link for students
+
+A list of profiles can be modified in the Profile repository. 
+
+To create a session link, add the name of the session after this url: `https://catcher-org.github.io/CATcher/?session=`. This profile name must correspond exactly to the profile name in Profile repository. 
```

Comment:
Should we mention that the instructor need to do url-encoding of the name of the session? Also should we put an example using something like `https://catcher-org.github.io/CATcher/?session=CS2103%2FT%20Alpha%20Test` for `"CS2103/T Alpha Test"`?

Reply:
Yes, good idea